homeassistant/components/rainmachine/diagnostics.py

```diff
@@ -42,13 +41,11 @@ async def async_get_config_entry_diagnostics(
     """Return diagnostics for a config entry."""
     data: RainMachineData = hass.data[DOMAIN][entry.entry_id]
 
-    controller_tasks = {
-        "versions": data.controller.api.versions(),
-        "current_diagnostics": data.controller.diagnostics.current(),
-    }
-    controller_results = await asyncio.gather(
-        *controller_tasks.values(), return_exceptions=True
-    )
+    try:
+        controller_diagnostics = await data.controller.diagnostics.current()
+    except RainMachineError:
+        LOGGER.warning("Unable to download controller-specific diagnostics")
+        controller_diagnostics = None
```

Comment:
Is this change intentional?

Reply:
Nope. Force-push coming.

Reply:
Roger: the versions data is now included in a coordinator, so we only need to the diagnostics one.